charts/prometheus-redis-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 1.11.1
+appVersion: 1.27.0
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 4.2.0
+version: 4.2.1
```

Comment:
@lbtm 
Could we bump the minor version please, e.g. 4.2.0 -> 4.3.0? It seems to me that the appVersion jumps quite a lot.